### PR TITLE
Added materialize icons

### PR DESCRIPTION
**Pull request by ujamer**
- [x] Added pollItem component
- [x] Added some custom styles for the poll items

### CODE REVIEW

File: client/styles/main.scss [hunk 1]
```diff
@@ -1 +1,9 @@
 @import "components/page-head";
+
+.collection-item .row {
```
Comment (ekawatani) [inline client/styles/main.scss]: These need to be extracted a separate file.

File: client/views/components/PollItem.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+PollItem = React.createClass({
+  render() {
+    var id = Random.id();
+    var divStyle = {
```
Comment (ekawatani) [inline client/views/components/PollItem.jsx]: Are this and id used? Seems like they aren't?

File: client/views/pages/New.jsx [hunk 3]
```diff
@@ -30,6 +30,14 @@ New = React.createClass({
...
               </div>
             </div>
+            <div className="divider" />
+            <div className="row">
+              <h5>Participents can select from these options:</h5>
+              <ul className="collection">
+                <PollItem />
+                <PollItem />
+              </ul>
+            </div>
```
Comment (ekawatani) [inline client/views/pages/New.jsx]: I guess you are going to extract this into a separate component? :p

File: client/views/components/PollItem.jsx (hunk 2 above)
Comment (ujamer) [inline client/views/components/PollItem.jsx]: Right, and I need to change the title to WIP too.

File: client/styles/main.scss (hunk 1 above)
Comment (ujamer) [inline client/styles/main.scss]: Will do that.

File: client/views/pages/New.jsx (hunk 3 above)
Comment (ujamer) [inline client/views/pages/New.jsx]: Yes, and read from the choices schema.

**Pull request merged**